pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -1991,6 +2038,35 @@
       ^-  ?
       &(=(item-binding binding) =(item-duct duct))
     ==
+  ::  +add-cache: remember a cache mapping
+  ::
+  ++  add-cache
+    |=  [=path =cache-entry]
+    ^-  [(list move) server-state]
+    =.  cache.state  (~(put of cache.state) path cache-entry)
+    `state
+  ::  +del-cache: forget a cache mapping
+  ::
+  ++  del-cache
+    |=  =path
```

Comment:
Similarly, deletion should remove the data inside the binding for the cache-entry, leaving a ~ to denote that data was once there but is no longer available.